torchaudio/backend/common.py

```diff
@@ -72,6 +89,10 @@ def __init__(self,
 
 _LOAD_DOCSTRING = r"""Loads an audio file from disk into a tensor
 
+Warnings:
```

Comment:
I think this is unnecessary, because it's duplicated with the warning statement and because we don't do this in any other library.